node/env0-deploy-cli.js

```diff
@@ -78,16 +78,9 @@ const getEnvironmentVariablesOptions = (environmentVariables) => {
   if (environmentVariables && environmentVariables.length > 0) {
     console.log('getting Environment Variables from options:', environmentVariables);
     environmentVariables.forEach(config => {
-      const configArray = config.split(/=(.+)/);
-      if (configArray.length === 3) {
-        result.push({
-          name: configArray[0],
-          value: configArray[1]
-        })
-      }
-      else {
-        throw new Error(`environmentVariables options are invalid`);
-      }
+      let [name, ...value] = config.split('=');
```

Comment:
@Roni I know it's mainly my code, but could you please add some unit tests here with the use case of an empty environment variable and case that there is a '=' in the value?

Reply:
A proper testbed for this module is missing.  
We'll need to do some refactoring here to extract a cli module exposing this util as right now this code is deeply tangled in the cli code.  
I think that's out of scope right now.